src/features/query-bar/__tests__/ValueField-test.js

```diff
@@ -72,5 +72,34 @@ describe('features/query-bar/components/filter/ValueField', () => {
                 expect(component.prop(valuePropNames[componentName])).toEqual(expectedValue);
             });
         });
+
+        describe('when there is an error due to an invalid input', () => {
+            describe('should not show an error for a', () => {
```

Comment:
I need help to understand from the user's pov. Why omit error when invalid input?

Reply:
We agreed to delete line 76.

Reply:
Discussed in person and decided that we can get rid of L76.